tfhe/docs/how_to/serialization.md

```diff
@@ -69,3 +69,92 @@ fn server_function(serialized_data: &[u8]) -> Result<Vec<u8>, Box<dyn std::error
     Ok(serialized_result)
 }
 ```
+
+
+# Safe Serialization/Deserialization
+
+For some types, safe serialization and deserialization functions are available.
+Bincode is used internally.
+
+Safe-deserialization must take as input the output of a safe-serialization.
+On this condition, validation of the following is done:
+- type: trying to deserialized `type A` from a serialized `type B` raises an error along the lines *On deserialization, expected type A, got type B* instead of an generic deserializtion error (or less likely a meaningless result of `type A`)
+- versions: trying to deserialized `type A` (version 0.2) from a serialized `type A` (incompatible version 0.1) raises an error along the lines *On deserialization, expected serialization version 0.2, got version 0.1* instead of an generic deserializtion error (or less likely a meaningless result of `type A` (version 0.2))
```

Comment:
trying to deserialized -> trying to deserialize

Reply:
instead of an generic -> instead of a generic

Reply:
along the lines -> along the lines of

Reply:
Fixed